src/libs/react-native-device-info/__mocks__/getUniqueId.web.ts

```diff
@@ -0,0 +1 @@
+export const getUniqueId = jest.fn().mockReturnValue('ad7b7b5a169641e27cadbdb35adad9c4ca23099a')
```

Comment:
We don't need `src/libs/react-native-device-info/__mocks__/getUniqueId.web.ts` if we have the exact same `src/libs/react-native-device-info/__mocks__/getUniqueId.ts`, do we?

Reply:
`yarn test:unit:web 'apiHelper`

```
 FAIL  src/api/__tests__/apiHelpers.test.ts (5.135 s)
  [api] helpers
    [method] safeFetch
      ✕ should call fetch with populated header (154 ms)
      ✕ should call fetch with populated header when route is in NotAuthenticatedCalls (52 ms)
      ✓ needs authentication response when refresh token fails (39 ms)
      ✕ regenerates the access token and fetch the real url after when the access token is expired (9 ms)
      ✓ needs authentication response when there is no access token (2 ms)
      ✓ needs authentication response when there is no refresh token (1 ms)
```

c'est malheureusement nécessaire, sinon les tests webs appellent la vraie implem